compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -978,6 +1002,25 @@ object Parsers {
         if lookahead.token == RBRACE then followingIsTypeStart() else recur()
       }
 
+    /** Under qualifiedTypes language import: is the following token sequence a
+     *  qualified type `<<< id: type with boolean >>>` ?
```

Comment:
Do `<<<` and `>>>` mean `(indent | '{') ` and `(dedent | '}')`? Seems to me like writing these explicitly would be clearer.

Reply:
Well not quite, see https://docs.scala-lang.org/scala3/reference/syntax.html#optional-braces for more information

Reply:
Ah, that makes much more sense with the documentation. Okay, let's keep it like that then!